### ISSUE TITLE
_optimize.pyx missing from tarball on pypi

### ISSUE THREAD

**Issue by dpryan79** [PyProphet/pyprophet#92]
The automated update on bioconda of pyprophet (https://github.com/bioconda/bioconda-recipes/pull/26899 ) is throwing an error:

```
11:32:42 BIOCONDA INFO (OUT) Processing $SRC_DIR
11:32:43 BIOCONDA INFO (OUT)     ERROR: Command errored out with exit status 1:
11:32:43 BIOCONDA INFO (OUT)      command: /opt/conda/conda-bld/pyprophet_1613734077926/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-4kgkzoca/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-4kgkzoca/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-d71avhe_
11:32:43 BIOCONDA INFO (OUT)          cwd: /tmp/pip-req-build-4kgkzoca/
11:32:43 BIOCONDA INFO (OUT)     Complete output (11 lines):
11:32:43 BIOCONDA INFO (OUT)     Traceback (most recent call last):
11:32:43 BIOCONDA INFO (OUT)       File "<string>", line 1, in <module>
11:32:43 BIOCONDA INFO (OUT)       File "/tmp/pip-req-build-4kgkzoca/setup.py", line 56, in <module>
11:32:43 BIOCONDA INFO (OUT)         ext_modules=cythonize(ext_modules, compiler_directives={'language_level' : "2"}),
11:32:43 BIOCONDA INFO (OUT)       File "/opt/conda/conda-bld/pyprophet_1613734077926/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/Cython/Build/Dependencies.py", line 972, in cythonize
11:32:43 BIOCONDA INFO (OUT)         aliases=aliases)
11:32:43 BIOCONDA INFO (OUT)       File "/opt/conda/conda-bld/pyprophet_1613734077926/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/Cython/Build/Dependencies.py", line 815, in create_extension_list
11:32:43 BIOCONDA INFO (OUT)         for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
11:32:43 BIOCONDA INFO (OUT)       File "/opt/conda/conda-bld/pyprophet_1613734077926/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/Cython/Build/Dependencies.py", line 114, in nonempty
11:32:43 BIOCONDA INFO (OUT)         raise ValueError(error_msg)
11:32:43 BIOCONDA INFO (OUT)     ValueError: 'pyprophet/_optimized.pyx' doesn't match any files
```

Inspecting the package, there is indeed no .pyx file, just the .c file produced by cython. This appears to related to #95 (specifically: https://github.com/PyProphet/pyprophet/pull/91/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R7 )

**Comment by grosenberger** [comment]
Thanks for the notice. This should now hopefully be fixed in PyProphet 2.1.10 on PyPI. Please let me know if this issue or others persist.

**Comment by dpryan79** [comment]
Seems fixed now, thanks!
